src/styles/index.scss

```diff
@@ -1,3 +1,111 @@
+@import "./variables";
+@import "./stars.scss";
+
+* {
+  box-sizing: border-box;
+}
+
 body {
```

Comment:
To specify styles, it is better to use classes as much as possible, and less to refer directly to tags.

Reply:
but what about coment from other mentor? if i have no not enough experience for understanding, so how should i do

Reply:
@AnastasiiaHombalevska To set styles for the body, you can create a class and set it for the body.
```
.container { ... }
...
<body class="container"> ... </body>
```
Or set these styles for the `product-card` class, since it is the parent of all other elements (personally, I would try this option).